### PR TITLE
Add strong antibiotic: destragon.

### PR DESCRIPTION
**Pull request by TheRafters**
#### Summary
```SUMMARY: Balance "Add a strong antibiotic."```

#### Purpose of change
Add a rare and unhealthy prescription drug to make use of the STRONG_ANTIBIOTIC flag. Changes the description of antibiotics to reflect their mid-tier status. I added it to drugs_rare, hospital_lab and hard_drugs item groups at lower rates than antibiotics. Also added antibiotics to hospital_lab because it seemed appropriate for the location.

#### Describe the solution
We've got the code for a stronger form of antibiotic, but it's not used anywhere. This adds a stronger form of antibiotic that affects the players health adversely when used, which is similar to the effects of potent antibiotics IRL. Antibiotics are unchanged aside from the description.

#### Describe alternatives you've considered

I did consider adding a minor health hit to "antibiotics" and removing the health hit from "weak_antibiotics"

#### Testing

Loaded and spawned the drug. Took it and noted the different message in @ screen. Open to suggestions for further testing.

#### Additional context

I originally intended to add a 'nausea' effect to strong_antibiotics, but the STRONG_ANTIBIOTICS effect overrides any additional effects added to the "use_action" field. It was suggested that an actual infection will provide all the nausea one could want, but that doesn't address the effects of the medication.

It was suggested that the current health system is still balanced for 6 second turns in conversation in discord, since health effects don't seem to last for very long unless they involve a health cap (XXXL start), or make use of the leukocyte breeder. I wouldn't know where to look to determine if this is true.

### CODE REVIEW

**Comment by Night-Pryanik** [comment]
It is intended it's called `Trioxycycline`? IRL it's called `Tryocycline`.

File: data/json/items/comestibles/med.json [hunk 1]
```diff
@@ -1437,5 +1437,23 @@
       "move_cost": 100,
       "effects": [ { "id": "pkill1", "duration": 720 } ]
     }
+  },
+  {
+    "id": "strong_antibiotic",
+    "type": "COMESTIBLE",
+    "comestible_type": "MED",
+    "name": "trioxycycline",
+    "name_plural": "trioxycyclines",
```
Comment (Night-Pryanik) [inline data/json/items/comestibles/med.json]: Redundant.

Comment (Malkeus) [reply data/json/items/comestibles/med.json]: I was intending to name it something non-existent, like Atreyupan. I'll remove the plural field though.

**Comment by Malkeus** [comment]
My intention was to name the drug something that doesn't exist IRL, similar to Atreyupan. Unless Atreyupan exists and doesn't show up on google searches? I'm open to suggestions for the name.

**Comment by kevingranade** [comment]
If it's not a real antibiotic don't give it a name similar to a real antibiotic, people are going to keep thinking it's mispelled.

**Comment by Malkeus** [comment]
That's what I was trying to do with my random mismash of prefixes. I can't find Tryocycline in a google search either. How about Dapterocozamin? I could go slang route and call it 'Burnout'. I suck at naming things...

**Comment by Malkeus** [comment]
The only objection seems to be the name. This is ready to go if the name is acceptable. I bumped the price up about 40%-ish as well. Drug companies gotta squeeze every drop out of the consumer after all.

**Pull request merged**